internal/pkg/cli/deploy/workload.go

```diff
@@ -68,7 +68,8 @@ func (noopActionRecommender) RecommendedActions() []string {
 	return nil
 }
 
-type imageBuilderPusher interface {
+type repositoryService interface {
+	Login(docker repository.ContainerLoginBuildPusher) (string, string, error)
```

Comment:
how much do we care about showing the output of the `docker login` command if it succeeds? if we don't care, we could remove the second string output and only include the output in the case of an error 🤔. just a thought - i'm happy either way!

Reply:
I thought it would be good to display that the `docker login` is successful.